### PR TITLE
Hotfix/rename

### PR DESCRIPTION
**Pull request by RWOverdijk**
chore(project): Renamed to swan-exampleHotfix/rename

### CODE REVIEW

**Comment by RWOverdijk** [comment]
@VMBindraban Review please

File: package.json [hunk 1]
```diff
@@ -31,6 +31,8 @@
   "main": "app.js",
   "repository": {
     "type": "git",
-    "url": "git://github.com/spoonx/aurelia-todo-server.git"
-  }
+    "url": "git://github.com/spoonx/swan-example-server.git"
+  },
+  "author": "rwoverdijk <wesley@spoonx.nl>",
```
Comment (VMBindraban) [inline package.json]: Author is already defined on line 14. Same for license on 15.

**Comment by RWOverdijk** [comment]
@VMBindraban Rereview please.

**Comment by VMBindraban** [comment]
:+1:

**Pull request merged**